spec/BasicSpec.js

```diff
@@ -1140,6 +1140,26 @@ describe('HtmlWebpackPlugin', function () {
       /<script type="text\/javascript" src="common_bundle.js">.+<script type="text\/javascript" src="util_bundle.js">.+<script type="text\/javascript" src="index_bundle.js">/], null, done);
   });
 
+  it('should sort the chunks in the order provided in none mode', function (done) {
+    testHtmlPlugin({
+      entry: {
+        util: path.join(__dirname, 'fixtures/util.js'),
+        index: path.join(__dirname, 'fixtures/index.js')
+      },
+      output: {
+        path: OUTPUT_DIR,
+        filename: '[name]_bundle.js'
+      },
+      plugins: [
+        new HtmlWebpackPlugin({
+          chunks: ['util', 'index'],
```

Comment:
Sorry but that's the whole point of my PR. If you leave out `chunks:['util','index']` the test will fail, with or without my code. It is not possible to specifying an explicit sort order ysing en entry config when it is specified as an object. Providing `entry: { 'util':.., 'index': ...}` and sortorder `none` will result in html where index is included before util.

I see the following options:

- Don't address it, requiring the caller to provide a sort function along the lines of what I quoted above. In that case we can just close this PR and move along.
- Address it by making the `chunks` option not only work as a filter but **also** as a way to specify the sort order. This is the solution I chose in this PR, and that is why the `chunks` option is part of the test because it specifies the sort order and that is what we are testing.
- Use `chunkSortMode` to specify an explicit sort order (`chunkSortModel: ['util','index']`)
- Other ideas?